documentation/slate/CONTRIBUTING.md

```diff
@@ -15,7 +15,12 @@ plus some Slate widgets (such as notices).
 
 Once some changes have been done, Slate needs to build HTML sources from Markdown files.
 
-First, install bundler using `apt-get install bundler` or `gem install bundler`.
+First, install bundler using `apt-get install bundler` or `gem install bundler`. Make sure you've also installed ruby's header to build the json package.
```

Comment:
Why you need to install ruby to run `/equipment_reports` API ?

Reply:
It's to generate the documentation